integration-tests/tests/run-notebook.js

```diff
@@ -40,7 +41,7 @@ const testRunNotebookFn = _.flow(
   // 500ms delay in the test is not enough to guarantee that the UI elements have finished moving.
   // Therefore, we start with a 1000ms delay, then make sure there are no spinners just in case the
```

Comment:
Nit: update comment to reflect new learning about the delay length.

Reply:
+1

Reply:
Addressed